### DISCUSSION TITLE
Mobility Data Specification-compatible? or another standard?

### DISCUSSION THREAD

**Discussion by kyrreso** [Norsk-institutt-for-samkjoring/Standard#5] [Ideas]
Would it be a good idea to make it compatible, for the [MDS](https://www.openmobilityfoundation.org/about-mds/) or should we try to get closer to another standard?

[ODS wiki](https://github.com/openmobilityfoundation/mobility-data-specification/wiki)

**Comment by MartinH-open** [comment]
Not sure what you mean by OM-compatible? Please reference that standard by some link.

**Reply by kyrreso** [reply]
Sorry, updated now. What other standards would be compatible with this standard?
